<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelFrame</class>
 <widget class="QWidget" name="PanelFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SectionContainer" name="sectionXYZ">
     <property name="title" stdset="0">
      <string>XYZ</string>
     </property>
     <widget class="QWidget" name="layoutXYZ">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionShape">
     <property name="title" stdset="0">
      <string>Shape</string>
     </property>
     <widget class="QWidget" name="layoutShape">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionFill">
     <property name="title" stdset="0">
      <string>Fill</string>
     </property>
     <widget class="QWidget" name="layoutFill">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionStroke">
     <property name="title" stdset="0">
      <string>Stroke</string>
     </property>
     <widget class="QWidget" name="layoutBorder">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionDropShadow">
     <property name="title" stdset="0">
      <string>Drop Shadow</string>
     </property>
     <widget class="QWidget" name="layoutDropShadow">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>sectioncontainer.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
